infrastructure/ansible/roles/ats/tasks/ats.yml

```diff
@@ -25,7 +25,12 @@
 - name: Deploy the udev device mapper script to a more reliable location
   # with /opt on separate lvm part, udev on boot may fail
   copy:
-    src: /opt/ort/supermicro_udev_mapper.pl
+    src: >
```

Comment:
See above whitespace and version quoting

Reply:
Fixed